--- gtk+-2.10.9/gdk/win32/gdkcursor-win32.c.orig	2007-01-22 16:59:17.000000000 +0100
+++ gtk+-2.10.9/gdk/win32/gdkcursor-win32.c	2007-03-08 11:18:25.259002577 +0100
@@ -27,7 +27,12 @@
 
 #include "xcursors.h"
 
-#if defined(__MINGW32__) || (defined(_MSC_VER) && (WINVER < 0x0500))
+#ifdef __MINGW32__
+/* get w32api version */
+#include <w32api.h>
+#endif
+
+#if (defined(__MINGW32__) && (__W32API_MAJOR_VERSION < 3 || __W32API_MINOR_VERSION < 8)) || (defined(_MSC_VER) && (WINVER < 0x0500))
 typedef struct { 
   DWORD        bV5Size; 
   LONG         bV5Width; 
