--- gtk+-2.24.5/gdk/win32/gdkdisplay-win32.c.orig	2011-06-18 10:42:04.488235257 +0200
+++ gtk+-2.24.5/gdk/win32/gdkdisplay-win32.c	2011-06-18 10:42:10.821568802 +0200
@@ -383,11 +383,13 @@
 
         GDK_NOTE (DND, g_print (" drawclipboard owner: %p", hwndOwner));
 
+#ifdef G_ENABLE_DEBUG
         if (_gdk_debug_flags & GDK_DEBUG_DND)
           {
             while ((nFormat = EnumClipboardFormats (nFormat)) != 0)
               g_print ("%s ", _gdk_win32_cf_to_string (nFormat));
           }
+#endif
 
         GDK_NOTE (DND, g_print (" \n"));
 
